build.sbt

```diff
@@ -1,11 +1,11 @@
 name := "derevo"
 import com.typesafe.sbt.SbtGit.git
 
-val publishVersion = "0.11.6"
+val publishVersion = "0.12.0"
 
 val common = List(
   scalaVersion := "2.13.4",
-  crossScalaVersions := List("2.12.12", "2.13.4"),
+  crossScalaVersions := List("2.12.13", "2.13.4"),
```

Comment:
and here, please
https://github.com/tofu-tf/derevo/blob/3dfca42dcb172edc370124456faf19e8741cde2d/.github/workflows/scala.yml#L45